setup.py

```diff
@@ -72,6 +72,7 @@
     'adage==0.8.5',
     'packtivity==0.10.0',
     'yadage==0.13.5',
+    'psycopg2-binary>=2.6.1',
```

Comment:
I think the PostgreSQL dependencies will be coming indirectly via `reana-commons`... So perhaps you can already start to proactively declare `reana-commons` as a dependency in `setup.py`, and we would just modify `Dockerfile` to pip-install it from github via `requirements-dev.txt` until the times `reana-commons` is released on PyPI... Seems more forward-friendly?

Reply:
Yes exactly, and that's a good idea, I'll go with this